lib/si-rabbitmq/src/environment.rs

```diff
@@ -1,10 +1,14 @@
 use rabbitmq_stream_client::error::{StreamCreateError, StreamDeleteError};
 use rabbitmq_stream_client::types::{ByteCapacity, ResponseCode};
 use rabbitmq_stream_client::Environment as UpstreamEnvironment;
+use std::time::Duration;
+use telemetry::prelude::{info, trace, warn};
 
-use crate::{config::Config, error::RabbitResult};
+use crate::{config::Config, error::RabbitResult, RabbitError};
 
 const STREAM_LENGTH_CAPACTIY_IN_MEGABYTES: u64 = 10;
+const ENVIRONMENT_CREATION_TIMEOUT_DURATION_IN_SECONDS: u64 = 4;
```

Comment:
It takes ~3-4 for the "Streams" service/feature to be ready. It's unclear if this is platform specific (i.e. macOS aarch64, resources specific (i.e. too few cores and RAM allocated), and/or feature specific (i.e. solely "Streams" or other RabbitMQ features too).